libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker-text.directive.ts

```diff
@@ -40,13 +40,16 @@ export class SkyColorpickerTextDirective {
     if (
       !isNaN(elementValue) &&
       elementValue >= 0 &&
+      this.maxRange &&
       elementValue <= this.maxRange
     ) {
       this.newColorContrast.emit({
-        color: this.color,
+        // This code assumed non-null pre-strict mode.
```

Comment:
Would this constitute a "TODO" comment? Something we need to address at some point?